### PR TITLE
Introduce NotionReminder for Notion Collections

### PR DESCRIPTION
**Pull request by xzavrel**
This adds new NotionReminder class aligned with NotionDate so it's possible to create and parse reminders in Collections. I have added also very brief example.

### CODE REVIEW

File: notion/collection.py [hunk 1]
```diff
@@ -465,11 +507,14 @@ def _convert_python_to_notion(self, val, prop, identifier="<unknown>"):
         if prop["type"] in ["title", "text"]:
             if not val:
                 val = ""
-            if not isinstance(val, str):
+            if not isinstance(val, (str,list)):
```
Comment (jamalex) [inline notion/collection.py]: Nit: space after commas (but I'll do a pass with `black` after which will clean things up).

Comment (jamalex) [reply notion/collection.py]: Also, I totally forgot that you can pass a tuple of types to `isinstance` -- nice!

Comment (jamalex) [reply notion/collection.py]: Can you clarify the case in which we'd be passing in a list here, though? Is this for when you'd want to use the native/internal notion format externally?

File: setup.py [hunk 2]
```diff
@@ -14,12 +14,12 @@
...
+    author="Vojtech Zavrel",
+    author_email="info@vojtechzavrel.cz",
     description="Unofficial Python API client for Notion.so",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/jamalex/notion-py",
+    url="https://github.com/zavrel/notion-py",
     install_requires=install_requires,
     include_package_data=True,
     packages=setuptools.find_packages(),
```
Comment (jamalex) [inline setup.py]: Could you revert the edits in this file? These are what get used to push to the main PyPI project, which should refer back here. Thanks!

Comment (xzavrel) [reply setup.py]: Sorry for that I thougth I made this pull request from feature branch, not from master.

File: notion/collection.py (hunk 1 above)
Comment (xzavrel) [reply notion/collection.py]: Same situation I didn'ŧ realize that I made pull request from master. Maybe I will do another pull request for this feature. But the main idea is, that I want to pass format that is not representable by markdown - e,g, text and backgroud colors, user mentions, dates etc. `[text, ['h', color]]`, `[' @' + client.get_user(user_id).full_name, [['b']]]`

**Comment by jamalex** [comment]
Thanks again for the proposed changes! Ended up merging https://github.com/jamalex/notion-py/pull/239, a slightly simpler approach to handling reminders. Let me know if that doesn't work for your needs.

**Pull request closed without merging**